pkg/devfile/adapters/interface.go

```diff
@@ -1,5 +1,7 @@
 package adapters
 
+import "io"
+
 type PlatformAdapter interface {
-	Start() error
+	Start(path string, out io.Writer, ignoredFiles []string, forceBuild bool, globExps []string, show bool) error
```

Comment:
I think we discussed renaming `Start` to `Push`. Should we still go ahead with that?

Reply:
Done